src/groupeddataframe/splitapplycombine.jl

```diff
@@ -263,8 +263,6 @@ const KWARG_PROCESSING_RULES =
     combine(fun::Union{Function, Type}, gd::GroupedDataFrame;
             keepkeys::Bool=true, ungroup::Bool=true)
     combine(pair::Pair, gd::GroupedDataFrame; keepkeys::Bool=true, ungroup::Bool=true)
-    combine(fun::Union{Function, Type}, df::AbstractDataFrame, ungroup::Bool=true)
```

Comment:
Is this documented somewhere else now?

Reply:
Yes - this is documented in `combine` for `AbstractDataFrame`. I removed it here because the signature was not correct (it had `ungroup::Bool=true` which in this case is not supported)